corehq/elastic.py

```diff
@@ -241,6 +242,26 @@ def iter_es_docs(index_name, ids):
                 yield result['_source']
 
 
+def iter_es_docs_from_query(query):
+    """Returns all docs which match query
+    """
+    scroll_result = query.scroll_ids()
+
+    def iter_export_docs():
+        with TransientTempfile() as temp_path:
+            with open(temp_path, 'w') as f:
```

Comment:
Can you use ```io.open``` instead of the python built-in ```open```?  The built-in changes between python 2 and 3, whereas ```io.open``` does not.

Reply:
:+1: